insect.desktop

```diff
@@ -0,0 +1,10 @@
+[Desktop Entry]
+Type=Application
+Name=Insect
+GenericName=Scientific calculator
+TryExec=insect
+Exec=insect
+Icon=insect
+Terminal=true
+Categories=Education;Science;Math;
+Keywords=science;math;numerical computation;calculator;physics;quantities;units
```

Comment:
There's another list of keywords in `web/index.html`:

```html
<meta name="keywords" content="calculator,scientific,math,physics,unit,conversion,quantity,SI,imperial,precision">
```

Should we update one list so that they're both identical, and if so, which list should we choose to use?